packages/plugin-experiment-browser/src/experiment.ts

```diff
@@ -0,0 +1,30 @@
+import { EnrichmentPlugin, BrowserClient, BrowserConfig } from '@amplitude/analytics-core';
+
+import {
+  ExperimentConfig,
+  Client as IExperimentClient,
+  initializeWithAmplitudeAnalytics,
+} from '@amplitude/experiment-js-client';
+
+export class ExperimentPlugin implements EnrichmentPlugin<BrowserClient, BrowserConfig> {
+  static pluginName = '@amplitude/experiment-analytics-plugin';
+  name = ExperimentPlugin.pluginName;
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  experiment: IExperimentClient;
+  config?: ExperimentConfig;
+
+  constructor(config?: ExperimentConfig) {
+    this.config = config;
+  }
+
+  async setup(config: BrowserConfig, _client: BrowserClient) {
+    this.experiment = initializeWithAmplitudeAnalytics(config.apiKey, this.config);
```

Comment:
@Mercy811 Experiment primarily uses deployment keys rather than the project API key to initialize the SDK. 

I will most likely need to be moved into the config, as for no deployment key we can fall back onto the project API key.

Reply:
Thanks @bgiori I fix it at https://github.com/amplitude/Amplitude-TypeScript/pull/1041